mlrun/common/schemas/alert.py

```diff
@@ -57,6 +58,7 @@ class EventKind(StrEnum):
     EventKind.SYSTEM_PERFORMANCE_SUSPECTED: [EventEntityKind.MODEL_ENDPOINT_RESULT],
     EventKind.MM_APP_ANOMALY_DETECTED: [EventEntityKind.MODEL_ENDPOINT_RESULT],
     EventKind.MM_APP_ANOMALY_SUSPECTED: [EventEntityKind.MODEL_ENDPOINT_RESULT],
+    EventKind.MM_APP_FAILED: [EventEntityKind.MODEL_ENDPOINT_RESULT],
```

Comment:
It would have been nicer to have a new type of `EventEntityKind` that would point at a MM app, rather than a result (which should conceptually be pointing at an app+result, IIUC). 
I see this type was already used for other types of events, so this is not a new thing, but still - is this something that was considered?
As a reference - when generating drift events which use the same type of entity, the `id` that is used is the result fqn, not just the app name.

Reply:
We considered it (@Yacouby ) and there were few concerns, mainly about confusing with `MODEL_ENDPOINT_RESULT` but your suggestion is indeed a proper approach. I'll add a new `EventEntityKind` called `MODEL_MONITORING_APPLICATION` which will handle only the new `mm_app_failure` event at the moment but in the future maybe we will use it for more events such as amount of app invocations, app performance indication, etc.